installation.md

```diff
@@ -91,34 +105,37 @@ For Rust to work properly, you'll need to have a compatible compiler suite insta
 * To use Nu with all possible optional features enabled, you'll also need the following:  
   ➜ on Linux (on Debian/Ubuntu): `apt install libxcb-composite0-dev libx11-dev`
 
-
 ## Installing from crates.io
 
-To install Nu via cargo (make sure you have installed [rustup](https://rustup.rs/) and the latest stable compiler via `rustup install stable`):
+To install Nu via cargo (make sure you have installed [rustup](https://rustup.rs/)
+and the latest stable compiler via `rustup install stable`):
 
 ```
 cargo install nu
 ```
 
-You can also install Nu with all the bells and whistles (be sure to have installed the [dependencies](https://book.nushell.sh/en/installation#dependencies) for your platform):
+You can also install Nu with all the bells and whistles (be sure to have installed the
+[dependencies](https://book.nushell.sh/en/installation#dependencies) for your platform):
 
 ```
-cargo install nu --all-features
+cargo install nu --features=stable
```

Comment:
@sebastian-xyz The [README](https://github.com/nushell/nushell#local) contains an almost identical passage to this one, but also includes the `--all` flag:

> You can also build Nu yourself with all the bells and whistles (be sure to have installed the dependencies for your platform), once you have checked out this repo with git:
>
>     cargo build --all --features=stable

Which is the correct command line?

Reply:
I'm not sure. According to the blog posts it is `--features=stable`. But I think `--all --features=stable` comes with more plugins.

@jonathandturner can you help us out?

Reply:
Yup, you got it. Building locally `--all --features=stable` gives you the most features and plugins

Reply:
Ok, thanks! I've added `--all` to the commands.